server/test/controllers/admin/AdminProgramStatusesControllerTest.java

```diff
@@ -262,7 +265,9 @@ public void update_editExistingStatus() throws ProgramNotFoundException {
 
     // Load the updated program and ensure the status is present.
     assertThat(
-            programService.getFullProgramDefinition(program.id).statusDefinitions().getStatuses())
+            repo.lookupActiveStatusDefinitions(
+                    programService.getFullProgramDefinition(program.id).adminName())
```

Comment:
**thought, non-blocking**: It seems like it'd be useful to have a method on the `ProgramService` that can map from program id to admin name.

Reply:
since getProgramDefinition() is calling the cached program, I am not too worried!

Reply:
It would be nice just to make it simpler for the various callers and readers of the code (less about the performance), but agree its nonblocking.